### ISSUE TITLE
[Bug] no Bottle o' Lightning

### ISSUE THREAD

**Issue by Adlez8475** [iron431/Irons-Spells-n-Spellbooks#594]
### Observed behaviour

When I right click on a charged creeper there is the sound and the glass bottle disappears but there is no Bottle o' Lightning in my inventory

### Expected behaviour

A Bottle o' Lightning should appears in my inventory when I right-click on a charged creeper.

### Steps to reproduce

1. find a charged creeper (or create one)
2. right-click on the creeper with an empty glass bottle
3. sound and the bottle disappears but no Bottle o' Lightning

### Server Type

Single Player

### Iron's Spells N Spellbooks version

irons_spellbooks-1.20.1-3.4.0.1.jar

### Forge version

120.1 - 47.3.0

### Other mods

I tested with only Iron's Spells 'n Spellbooks with nothing else and the issue still here.

**Comment by iron431** [comment]
i cannot reproduce this with just spells n spellbooks. are you sure youre using no other mods? are you in survival mode with inventory space, etc?

**Comment by Adlez8475** [comment]
Yes I'm sure, there is no other mods, only Iron's and its requirements. Here's a video I just shot to show you. It seems to work in creative mode but not in survival.
https://streamable.com/w6sj02

**Comment by iron431** [comment]
might be a forge version issue, ill have to look into it more
